src/parse_coin_config.c

```diff
@@ -1,6 +1,7 @@
+#include <string.h>
+#include <os.h>
+
 #include "parse_coin_config.h"
-#include "string.h"
-#include "os.h"
 
 typedef struct app_name_alias_s {
     char *foreign_name;
```

Comment:
const

Reply:
This comment applies to the fields of app_name_alias_s, github somehow placed it here